### PR TITLE
Open source guiding docs

### PR DESCRIPTION
**Pull request by andrewpbray**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Contributing
+
+You can contribute to Closeread in many ways:
+
+- By opening issues to provide feedback and share ideas.
+- By submitting Pull Request (PR) to fix opened issues
+- By submitting Pull Request (PR) to suggest new features (it is considered good practice to open an issue for discussion before working on a pull request for a new feature).
+
+Please be mindful of our [code of conduct](https://github.com/quarto-dev/quarto-cli/blob/main/.github/CODE_OF_CONDUCT.md) as you interact with other community members.
```
Comment (jimjam-slam) [inline CONTRIBUTING.md:9]: @andrewpbray This looks good! Happy to copy it over to ours

Comment (jimjam-slam) [reply CONTRIBUTING.md:9]: We may also need our email addresses, or a contact email, for reports

Comment (andrewpbray) [reply CONTRIBUTING.md]: Ah, I missed this ref to a c of c. I went ahead and struck that; we can put one in if the project grows.

**Comment by andrewpbray** [comment]
I think this one is ready for merging!

File: CONTRIBUTING.md (hunk 1 above)
Comment (jimjam-slam) [reply CONTRIBUTING.md]: Yeah, that's easier!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Contributing
+
+You can contribute to Closeread in many ways:
+
+- By opening issues to provide feedback and share ideas.
```
Comment (jimjam-slam) [inline CONTRIBUTING.md:5]: Will add 'and starting discussions' to this before I merge

**Review by jimjam-slam — APPROVED** [review]

**Pull request merged**